nsxt/data_source_nsxt_compute_manager.go

```diff
@@ -57,6 +55,11 @@ func dataSourceNsxtComputeManagerRead(d *schema.ResourceData, m interface{}) err
 		var perfectMatch []model.ComputeManager
 		var prefixMatch []model.ComputeManager
 		for _, objInList := range objList.Results {
+			if len(objName) == 0 {
+				// We want to grab single compute manager
+				perfectMatch = append(perfectMatch, objInList)
```

Comment:
So if there are multiple compute managers, an empty `display_name` will cause an error. I'm wondering if we should add a more informative err msg 
 indicating empty display_name can only be used with single compute manager to reduce the confusion or change the behavior here to just grab the first compute manager or just treat it as a match all for prefixMatch. What do you think?

Reply:
I don't think we should decide which manager to grab without further input from the user. 
Failing the data source because multiple objects are matching is a behavior present in most data sources in the provider. Regarding error messages - perhaps we should unify these messages across all data sources for consistency? 
I can add more info in the doc for the case of compute manager.